docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -71,14 +70,27 @@ flowchart
 
      subgraph auth[Auth Service Pool]
        ta1[Teleport Auth Service]
-       ta3[Teleport Auth Service]
+       ta2[Teleport Auth Service]
     end
 
     lb2-->auth
 
     auth-->kv[Key/Value Store]
     auth-->os[Object Store]
    end
+
+classDef teleportColor fill:#844cff,color:#fff
+classDef teleportColor2 fill:#a379ff,color:#fff
```

Comment:
At one point I had the idea to differentiate between Teleport Auth/Proxy and other Teleport services with colors. Happy to discuss if it's a good idea or not.

Reply:
Maybe it's just me/my monitor, but these colors look very similar to me! Should we just use one Teleport color? 

Personally, I'm in favor of leaving all styling of our diagrams to an external stylesheet that we'll prepare later, but we can go with this approach in the meantime if you think it makes the graph clearer.

Reply:
Agreed that it can be one color. I'm also in favor of an external style sheet and can remove the colors once it's created... but FWIW leaving one or two examples like this may give @roraback and team something to go on while building said stylesheets.